app/views/organizations/staff/default_pet_tasks/_search_results.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="justify-content-md-between mb-4 mb-xl-0 gx-3">
```

Comment:
@ErinClaudio why do we have two partials to render the results? Can't we use the original table?